internal/static/storage.go

```diff
@@ -25,6 +25,17 @@ const (
 	ObjectTypeStyling
 )
 
+func (o *ObjectType) String() string {
+	switch *o {
```

Comment:
```suggestion
func (o ObjectType) String() string {
	switch o {
```

Reply:
please update the test(s)

and you'll have to fix the const above as i've forgot to type them (ObjectTypeUserAvatar ObjectType  = iota)